io.trustable.stpadocumentationtool.json

```
@@ -1,12 +1,12 @@
 {
     "app-id": "io.trustable.stpadocumentationtool",
     "runtime": "org.kde.Platform",
-    "runtime-version": "5.12",
+    "runtime-version": "5.15",
     "sdk": "org.kde.Sdk",
     "rename-icon": "stpa_documentation_tool",
     "rename-desktop-file": "stpa_documentation_tool.desktop",
     "rename-appdata-file": "stpa_documentation_tool.appdata.xml",
-    "finish-args": [ "--socket=x11",
+    "finish-args": [ "--socket=x11", "--socket=wayland", "--device=dri",
```

If this really support wayland, then: 

```suggestion
    "finish-args": [ "--socket=fallback-x11", "--socket=wayland", "--device=dri",
```

Also usually you want `--share=ipc` for X11 anyway.